src/Azure/Azure.Quantum.Client/Authentication/TokenFileContent.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+namespace Microsoft.Azure.Quantum.Authentication
+{
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// Contract for the contents of the token file used in TokenFileCredential.
+    /// </summary>
+    public class TokenFileContent
+    {
+        /// <summary>
+        /// The access token.
+        /// </summary>
+        [JsonPropertyName("access_token")]
+        public string? AccessToken { get; set; }
```

Comment:
Yes, unfortunately, the compiler will complain about properties that don't have an initial value if using nullable directive. In this case, if I dropped the "?", it would require that we add something like = "" after the { get; set; }